src/lib/primitives/jsonschema/jsonschema.ts

```diff
@@ -289,7 +289,6 @@ function walkJsonschema({
         const [refName, reference] = references.get(childRef)!
         return $ref({
             ...context,
-            ...annotate(node, context),
```

Comment:
this removed documentation and non functional descriptions as well, do we want that?